packages/ds-medicare-gov/src/components/Dialog/index.js

```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+import { Dialog, CloseIconThin } from '@cmsgov/design-system';
+
+Dialog.defaultProps = {
+  ...Dialog.defaultProps,
+  closeButtonText: null,
```

Comment:
I've always thought this is kind of an accessibility issue 😕. At least it's really obvious that this feature is being removed, so we can address it in the future.